telco-hub/configuration/reference-crs/optional/lso/lsoNS.yaml

```diff
@@ -3,5 +3,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: openshift-local-storage
+  annotations:
+    argocd.argoproj.io/sync-wave: "-5"
```

Comment:
Without an annotation this is in wave 0 which should be early enough (it is before wave 2 where the localvolume is created.

Reply:
I have been trying to add some more order to the deployment, without modifying (too much) the existing one. This is I started to use negative numbers. 
Having more control on waves should allow us to deploy without many many re-syncs. But it is know this will require more extra work.